### PR TITLE
Fixed Linting Issues in src/appengine

### PR DESCRIPTION
**Pull request by iamarshsingh**
Fixing Linting issues as mentioned in #1824. This PR Fixes all Linting issues in appengine.

These are the fixed issues:
************* Module src.appengine.handlers.base_handler
handlers/base_handler.py:63:4: R1705: Unnecessary "elif" after "return" (no-else-return)
handlers/base_handler.py:246:11: R1714: Consider merging these comparisons with "in" to 'status in (403, 401)' (consider-using-in)
************* Module src.appengine.handlers.fuzzer_stats
handlers/fuzzer_stats.py:76:5: R1714: Consider merging these comparisons with "in" to "typename in ('integer', 'float')" (consider-using-in)
handlers/fuzzer_stats.py:87:5: R1714: Consider merging these comparisons with "in" to 'type_value in (int, float)' (consider-using-in)
handlers/fuzzer_stats.py:151:2: R1705: Unnecessary "elif" after "return" (no-else-return)
handlers/fuzzer_stats.py:198:20: W1505: Using deprecated method escape() (deprecated-method)
handlers/fuzzer_stats.py:447:10: R1718: Consider using a set comprehension (consider-using-set-comprehension)
************* Module src.appengine.handlers.cron.manage_vms
handlers/cron/manage_vms.py:377:6: W1505: Using deprecated method warn() (deprecated-method)
handlers/cron/manage_vms.py:397:27: R1718: Consider using a set comprehension (consider-using-set-comprehension)
handlers/cron/manage_vms.py:508:8: W1505: Using deprecated method warn() (deprecated-method)
************* Module src.appengine.handlers.testcase_detail.testcase_variants
handlers/testcase_detail/testcase_variants.py:28:6: R1705: Unnecessary "elif" after "return" (no-else-return)
************* Module src.appengine.libs.filters
libs/filters.py:59:2: R1705: Unnecessary "elif" after "return" (no-else-return)
************* Module src.appengine.libs.helpers
libs/helpers.py:34:2: W0107: Unnecessary pass statement (unnecessary-pass)
libs/helpers.py:146:2: R1705: Unnecessary "else" after "return" (no-else-return)
************* Module src.appengine.libs.issue_management.jira
libs/issue_management/jira/__init__.py:25:0: W0223: Method 'actions' is abstract in class 'Issue' but is not overridden (abstract-method)
libs/issue_management/jira/__init__.py:25:0: W0223: Method 'merged_into' is abstract in class 'Issue' but is not overridden (abstract-method)
libs/issue_management/jira/__init__.py:116:2: W0221: Parameters differ from overridden 'save' method (arguments-differ)
libs/issue_management/jira/__init__.py:139:0: W0223: Method 'find_issues_url' is abstract in class 'IssueTracker' but is not overridden (abstract-method)
************* Module src.appengine.libs.issue_management.monorail
libs/issue_management/monorail/__init__.py:22:0: C0414: Import alias does not rename original package (useless-import-alias)
************* Module src.appengine.libs.issue_management.monorail.credential_storage
libs/issue_management/monorail/credential_storage.py:25:2: C0111: Missing method docstring (missing-docstring)
libs/issue_management/monorail/credential_storage.py:38:23: W0613: Unused argument 'credentials' (unused-argument)
************* Module src.appengine.libs.query.datastore_query
libs/query/datastore_query.py:196:11: R1718: Consider using a set comprehension (consider-using-set-comprehension)

These issues are not fixed as @oliverchang mentioned that `appengine_config.py` will be deleted.
************* Module src.appengine.appengine_config
appengine_config.py:52:2: E0602: Undefined variable 'reload' (undefined-variable)
appengine_config.py:66:0: E0602: Undefined variable 'reload' (undefined-variable)

Please let me know if there are any changes required. Also, should I fix linting issues in other parts of the database and add it to this PR or make different PRs for different parts of codebase?

### CODE REVIEW

File: src/appengine/libs/issue_management/jira/__init__.py [hunk 1]
```diff
@@ -135,6 +135,14 @@ def save(self):
 
     self.itm.save(self)
 
+  def actions(self):
+    # To prevent abstract exception.
+    return
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/jira/__init__.py:140]: probably just "pass".

File: src/appengine/libs/issue_management/jira/__init__.py [hunk 2]
```diff
@@ -135,6 +135,14 @@ def save(self):
...
+
+  def merged_into(self):
+    # To prevent abstract exception.
+    return
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/jira/__init__.py:144]: probably just "pass".

File: src/appengine/libs/issue_management/jira/__init__.py [hunk 3]
```diff
@@ -172,6 +180,10 @@ def issue_url(self, issue_id):
     url = config.jira_url + '/browse/' + str(issue_id)
     return url
 
+  def find_issues_url(self, keywords=None, only_open=None):  # pylint: disable=unused-argument
+    # To prevent abstract exception.
+    return
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/jira/__init__.py:185]: probably just "pass".

File: src/appengine/libs/issue_management/monorail/credential_storage.py [hunk 4]
```diff
@@ -23,6 +23,7 @@ class CredentialStorage(Storage):
   """Instead of reading a file, just parse a config entry."""
 
   def locked_get(self):
+    """ TODO ADD DOC-STRING"""
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/monorail/credential_storage.py:26]: Add a docstring.
Return credentials ?

File: src/appengine/libs/issue_management/jira/__init__.py [hunk 5]
```diff
@@ -172,6 +180,10 @@ def issue_url(self, issue_id):
     url = config.jira_url + '/browse/' + str(issue_id)
     return url
 
+  def find_issues_url(self, keywords=None, only_open=None):  # pylint: disable=unused-argument
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/jira/__init__.py:183]: Add a "FIXME: Add support for keywords and only_open arguments" at end of function.

File: src/appengine/libs/issue_management/jira/__init__.py [hunk 6]
```diff
@@ -113,7 +113,7 @@ def components(self):
     """The issue component list."""
     return self._components
 
-  def save(self):
+  def save(self, new_comment=None, notify=True):  # pylint: disable=unused-argument
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/jira/__init__.py:116]: Add "FIXME: Add support for new_comment and notify arguments".

File: src/appengine/libs/issue_management/jira/__init__.py (hunk 1 above)
Comment (iamarshsingh) [reply src/appengine/libs/issue_management/jira/__init__.py:140]: pass was also giving linting issues, any other alternative? 
Also, I just removed a pass statement in another function [here](https://github.com/google/clusterfuzz/pull/1832/files#diff-c61315aee0223e7d4d4767c339334fb7L34)

File: src/appengine/libs/issue_management/monorail/credential_storage.py (hunk 4 above)
Comment (iamarshsingh) [reply src/appengine/libs/issue_management/monorail/credential_storage.py:26]: Sure, will add it.

File: src/appengine/libs/issue_management/jira/__init__.py (hunk 6 above)
Comment (iamarshsingh) [reply src/appengine/libs/issue_management/jira/__init__.py:117]: Added, please check.

File: src/appengine/libs/issue_management/jira/__init__.py (hunk 5 above)
Comment (iamarshsingh) [reply src/appengine/libs/issue_management/jira/__init__.py:183]: Added, please check.

File: src/appengine/libs/issue_management/monorail/credential_storage.py (hunk 4 above)
Comment (iamarshsingh) [reply src/appengine/libs/issue_management/monorail/credential_storage.py]: Added, please check.

File: src/appengine/libs/issue_management/jira/__init__.py (hunk 3 above)
Comment (iamarshsingh) [reply src/appengine/libs/issue_management/jira/__init__.py]: Added, please check.

File: src/appengine/libs/issue_management/jira/__init__.py (hunk 2 above)
Comment (iamarshsingh) [reply src/appengine/libs/issue_management/jira/__init__.py]: Added, please check.

File: src/appengine/libs/issue_management/jira/__init__.py (hunk 1 above)
Comment (iamarshsingh) [reply src/appengine/libs/issue_management/jira/__init__.py]: Added, please check.

**Comment by oliverchang** [comment]
Thanks @iamarshsingh!

**Pull request merged**